docs/proposals/implement-api-that-imports-prometheus-blocks-into-cortex.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Implement API that imports Prometheus blocks into Cortex"
```

Comment:
Let's make a better title :)
Like `tools to backfill TSDB blocks`?
Btw will we support Thanos blocks?

Reply:
Yeah, I have made the suggested changes. Can you please take a look at it? 🙇🏻 @yeya24 @alanprot

Reply:
Hi @alanprot 👋🏻 
I have made some changes in this, PR. Can you please review it 🙇🏻 🙏🏻